types/lyricist/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove all the rules here and fix errors, new packages cannot have anything except extends rule in tslint json file